### PR TITLE
Update button styles

### PR DESCRIPTION
**Pull request by nmcclay**

### CODE REVIEW

**Comment by nmcclay** [comment]
screenshot 2015-09-05 14 19 10

File: app/assets/stylesheets/forever_style_guide/modules/_button.scss [hunk 1]
```diff
@@ -156,6 +137,7 @@
     height: $btn-sm-size;
     padding: $padding-small-vertical $btn-padding-small-horizontal;
     border-radius: $border-radius-small;
+    line-height: 1.6;
```
Comment (schweitzdesign) [inline app/assets/stylesheets/forever_style_guide/modules/_button.scss]: line-height should precede border-radius in declaration order

File: app/assets/stylesheets/forever_style_guide/modules/_button.scss [hunk 2]
```diff
@@ -166,6 +148,7 @@
     font-size: font-size('small');
     height: $btn-xs-size;
```
Comment (schweitzdesign) [inline app/assets/stylesheets/forever_style_guide/modules/_button.scss]: declaration order

File: app/assets/stylesheets/forever_style_guide/modules/_button.scss [hunk 3]
```diff
@@ -47,38 +49,17 @@
...
-      @include btn-color('warning', true);
-    }
-  }
-  &.btn-danger {
-    @include btn-color('danger');
-    &.btn-inverse {
-      @include btn-color('danger', true);
+
+  // creates a button style for each color in color dictionary
+  @each $id, $color in $colors {
```
Comment (schweitzdesign) [inline app/assets/stylesheets/forever_style_guide/modules/_button.scss]: Here's a sample of the errors I'm encountering when running this against the store. It looks like somewhere between this change, the semantic colors and the color_utils, something's not quite right.

screen shot 2015-09-08 at 9 17 39 am

**Comment by schweitzdesign** [comment]
Looks good. I'm very much in favor of the simplified color effects.

**Pull request merged**